.github/release-drafter.yml

```diff
@@ -0,0 +1,40 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+template: |
+  # Changelog
+  $CHANGES
+  
+  See details of [all code changes](https://github.com/altinn/altinn-access-management-frontend/compare/$PREVIOUS_TAG...v$RESOLVED_VERSION) since previous release 
+  
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+  - title: '🧰 Maintenance'
+    labels:
+      - 'infrastructure'
+      - 'automation'
+      - 'documentation'
+      - 'dependencies'
+  - title: '🏎 Performance'
+    label: 'performance'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+version-resolver:
```

Comment:
As discussed major, minor and patch version does not make much sense in our context of access-management-frontend container app. But could instead just have: {release number}.{year}

Reply:
A new PR is created :)